docs/prepare-a-shared-rp-development-environment.md

```diff
@@ -230,9 +227,9 @@ locations.
    ```
 
    ```bash
+   // note: for macos change the -w0 option for base64 to -b0
```

Comment:
IMO you could add these notes like we do in other parts of the docs so that they are visible:
```suggestion
   > __NOTE:__: for macos change the -w0 option for base64 to -b0
```